README.md

```diff
@@ -17,6 +17,17 @@ cd my-app/
 npm start
 ```
 
+## Node_module packages that also need typescript processing
+To enable typescript processing on node_module folders (npm packages) you may add the following environment variable to your package.
+```sh
+REACT_APP_TYPESCRIPT_NODE_MODULES_FOLDERS="$$folder name(s) space delimited$$"
+```
+Ex. for internal, namespaced packages like `{root}/node_modules/@company-name/unique` you could set
+```sh
+REACT_APP_TYPESCRIPT_NODE_MODULES_FOLDERS="@company-name"
+```
+
+
```

Comment:
I'd say this is a bit too complicated. Importing `ts` source files from `node` packages is more of an exceptional than regular behavior. Extending the `include` clause of the `tsx?` rule to cover the `node_modules` folder is more simple and does not hurt anyone not using it, since it is only applied in this very specific case.

Reply:
Are you sure? I feel like that would slow down a large project during dev builds; to process every package...
I think what may be best is us maintaining a fork for ourselves, and maybe I will write a medium article on what I changed for anyone else that wants to do the same; since it is a pretty limited use-case.
I don't want to decrease dev performance for all users of this package just to meet our niche requirement.